VTUL/dlp-access
# Update packages and adds Authenticator component

**Update packages and adds Authenticator component.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2616) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Update packages and adds Authenticator component

# What's the changes? (:star:)
* Update packages and adds Authenticator component
* Refactors admin tests for cleaner AWS authentication

# How should this be tested?
* Run tests
* Check admin authentication

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.
* branch: `react_amplify_update`

# Interested parties
@yinlinchen 

(:star:) Required fields
